Assets/Plugins/iOS/Shopify/Checkout/Checkout.WebViewController.swift

```diff
@@ -116,4 +151,17 @@ extension Checkout.WebViewController: WKNavigationDelegate {
     private func isThankYouPage(_ url: URL) -> Bool {
         return url.absoluteString.hasSuffix("/thank_you")
     }
+    
+    private func handleLoadError(_ error: NSError) {
+        let message: String
+        switch error.code {
```

Comment:
This `switch` might be better suited as an `extension` on `NSError`. Something like:
```swift
extension NSError {
    var networkErrorDescription: String {
        // swift statement
    }
}
```